src/app/public/modules/description-list/description-list-adapter-service.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+/**
+ * @internal
+ */
+@Injectable()
+export class SkyDescriptionListAdapterService {
+
+  private renderer: Renderer2;
+
+  constructor(
+    private rendererFactory: RendererFactory2
+  ) {
+    this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
+  }
+
+  public getWidth(elementRef: ElementRef): number {
+    return elementRef.nativeElement.clientWidth;
+  }
+
+  public setResponsiveClass(element: ElementRef): void {
```

Comment:
Just a question: I'm assuming this is duplicated from Trevor's responsive container utility?

Reply:
Yeah, unfortunately this component doesn't use the same breakpoints. There may be a way to extend what we have in `core` to handle different breakpoints, if you think that would be more worthwhile?

Reply:
Nah, that's fine. Why do we have different breakpoints for this component? Design's aware of this difference?

Reply:
Yes, it was part of the design spec. Another difference worth mentioning is these breakpoints are dependent on the parent element - not the window.